Gems/ROS2/Code/Source/Frame/ROS2FrameComponent.cpp

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2/Frame/ROS2FrameComponent.h"
+#include "ROS2/ROS2Bus.h"
+#include "ROS2/ROS2GemUtilities.h"
+#include "ROS2/Utilities/ROS2Names.h"
```

Comment:
angle brackets should be preferred here: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#including-headers
```suggestion
#include <ROS2/Frame/ROS2FrameComponent.h>
#include <ROS2/ROS2Bus.h>
#include <ROS2/ROS2GemUtilities.h>
#include <ROS2/Utilities/ROS2Names.h>
```